docs/quickstart_guide.md

```diff
@@ -307,10 +307,14 @@ for benchmarking asynchronous (i.e., Knative Eventing) case and more details abo
 ### 1. Deploy Functions
 **On the master node**, execute the following instructions below using **bash**:
 
-1. Optionally, configure the types and the number of functions to deploy in `examples/deployer/functions.json`.
-2. Run the deployer client:
+1. Clone vSwarm repo:
     ```bash
-    source /etc/profile && pushd ./examples/deployer && go build && popd && ./examples/deployer/deployer
+    git clone --depth=1 https://github.com/vhive-serverless/vSwarm ../vswarm
```

Comment:
I suggest preparing binaries of the invoker and deployer tools (e.g., by attaching them to a vSwarm release) and getting them via `wget` in this guide. otherwise we may not be able to track changes in vSwarm that may lead to failures in vSwarm

Reply:
May be fixing the version of vSwarm repo cloned would be better? For running it on different platforms you will need to build it on the spot.

Reply:
that would work. please ask David to prepare a release v1.0 for vSwarm